src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -159,6 +159,9 @@ def __init__(self, backend, x509):
         self._backend = backend
         self._x509 = x509
 
+    def __repr__(self):
+        return "<Certificate(subject={0})>".format(self.subject)
```

Comment:
One last small thing, can you make this: `subject={0}, ...` (with a literal ellipsis), that way it's clear there's other properties